geopandas/array.py

```diff
@@ -566,6 +566,12 @@ def geom_equals_exact(self, other, tolerance):
         return self._binary_method("equals_exact", self, other, tolerance=tolerance)
 
     def geom_almost_equals(self, other, decimal):
+        warnings.warn(
+            "geom_almost_equals() is now geom_equals_exact(). "
+            "geom_almost_equals() will be deprecated in the future.",
```

Comment:
I think it would make sense to be more explicit here (borrowing from the underlying shapely warning)
> "The 'geom_almost_equals()' method is deprecated because the name is
            confusing. The 'equals_exact()' method should be used
            instead."

Reply:
done.